docs/overview.md

```diff
@@ -3,3 +3,113 @@ id: overview
 title: Overview
 sidebar_label: Overview
 ---
+
+Eik is an [asset server](/docs/server) for serving ESM and CSS assets. The Eik server can run and be integrated with your own infrastructure. Eik comes with a [client](/docs/client) for easy upload and management of your assets.
+
```

Comment:
Do we want to mention that other assets (eg. SVG) can also be served or keep the scope narrow?

Reply:
Yeah. But I think that is more suitable in a more "extensive overview". We also need to explain the difference between the `npm` and `pkg` namespace.